[2024-06-03T17:38:10.853+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:38:10.868+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-03T17:38:10.874+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-03T17:38:10.874+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:38:10.883+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2019-03-01 00:00:00+00:00
[2024-06-03T17:38:10.888+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=79) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:38:10.889+0000] {standard_task_runner.py:63} INFO - Started process 83 to run task
[2024-06-03T17:38:10.889+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpazqafst6']
[2024-06-03T17:38:10.890+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask games_etl
[2024-06-03T17:38:10.923+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [running]> on host a5a4421409ea
[2024-06-03T17:38:10.983+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-03-01T00:00:00+00:00'
[2024-06-03T17:38:10.984+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:38:10.985+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:38:11.295+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:38:11.301+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_dag.py", line 19, in games_etl_task
    return get_monthly_games(execution_date)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_etl_code.py", line 60, in get_monthly_games
    connection = BaseHook.get_connection('chess_project_s3')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `chess_project_s3` isn't defined
[2024-06-03T17:38:11.307+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2019-03-01T00:00:00+00:00, execution_date=20190301T000000, start_date=20240603T173810, end_date=20240603T173811
[2024-06-03T17:38:11.316+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 11 for task games_etl (The conn_id `chess_project_s3` isn't defined; 83)
[2024-06-03T17:38:11.343+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-03T17:38:11.360+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:38:11.364+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-03T17:44:32.358+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:44:32.371+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-03T17:44:32.376+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-03T17:44:32.377+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:44:32.385+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2019-03-01 00:00:00+00:00
[2024-06-03T17:44:32.390+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=75) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:44:32.391+0000] {standard_task_runner.py:63} INFO - Started process 79 to run task
[2024-06-03T17:44:32.391+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp750uf2gp']
[2024-06-03T17:44:32.392+0000] {standard_task_runner.py:91} INFO - Job 68: Subtask games_etl
[2024-06-03T17:44:32.422+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T17:44:32.480+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-03-01T00:00:00+00:00'
[2024-06-03T17:44:32.482+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:44:32.482+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:44:32.741+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T17:44:33.277+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T17:44:33.278+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:44:33.287+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2019-03-01T00:00:00+00:00, execution_date=20190301T000000, start_date=20240603T174432, end_date=20240603T174433
[2024-06-03T17:44:33.326+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T17:44:33.344+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:44:33.346+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T14:09:58.488+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T14:09:58.503+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-08T14:09:58.510+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-08T14:09:58.510+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T14:09:58.522+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2019-03-01 00:00:00+00:00
[2024-06-08T14:09:58.527+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=91) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T14:09:58.529+0000] {standard_task_runner.py:63} INFO - Started process 93 to run task
[2024-06-08T14:09:58.529+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '208', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp4d1rh4d7']
[2024-06-08T14:09:58.530+0000] {standard_task_runner.py:91} INFO - Job 208: Subtask games_etl
[2024-06-08T14:09:58.566+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [running]> on host 3c9d89bfdaf1
[2024-06-08T14:09:58.639+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-03-01T00:00:00+00:00'
[2024-06-08T14:09:58.640+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T14:09:58.641+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T14:09:58.964+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T14:09:59.488+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T14:09:59.489+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T14:09:59.495+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2019-03-01T00:00:00+00:00, execution_date=20190301T000000, start_date=20240608T140958, end_date=20240608T140959
[2024-06-08T14:09:59.545+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T14:09:59.558+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-08T14:09:59.559+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T14:51:27.886+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T14:51:27.901+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-08T14:51:27.909+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2024-06-08T14:51:27.909+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T14:51:27.921+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2019-03-01 00:00:00+00:00
[2024-06-08T14:51:27.926+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=201) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T14:51:27.928+0000] {standard_task_runner.py:63} INFO - Started process 212 to run task
[2024-06-08T14:51:27.928+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '244', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpdclmnayd']
[2024-06-08T14:51:27.929+0000] {standard_task_runner.py:91} INFO - Job 244: Subtask games_etl
[2024-06-08T14:51:27.963+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2019-03-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T14:51:28.025+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-03-01T00:00:00+00:00'
[2024-06-08T14:51:28.027+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T14:51:28.028+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T14:51:28.257+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T14:51:28.819+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T14:51:28.820+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T14:51:28.828+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2019-03-01T00:00:00+00:00, execution_date=20190301T000000, start_date=20240608T145127, end_date=20240608T145128
[2024-06-08T14:51:28.863+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T14:51:28.879+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-08T14:51:28.881+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
